app/(authenticated)/dashboard/_components/profiles.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+interface Person {
+  id: number;
+  initials: string;
+  name: string;
+  role: string;
+  username: string;
+  totalLeads: number;
+}
+export default function profiles({ Leaderboard }:any) {
```

Comment:
Component's first letter should appear in Uppercase and props first letter should appear in Lowercase. So here we should use `Profiles` and `leaderboard` instead of `profiles` and `Leaderboard` respectively. Also your code is not well formatted, install **Prettier** extension in VS Code.

Reply:
fixed